container/cypress/e2e/test-app/iframe/iframe-container.cy.js

```diff
@@ -23,6 +23,32 @@ describe('Iframe Container Test', () => {
       });
   });
 
+  it('sendCustomMessage', () => {
+    cy.get(containerSelector)
+      .shadow()
+      .get('iframe')
+      .then(iframe => {
+        const $body = iframe.contents().find('body')
+        cy.wrap($body)
+          .find('#content')
+          .should('have.text', ' ');
```

Comment:
whats the idea behind this empty string text check ?

Reply:
I wanted to check the states before and after sendCustomMessage() but you're right, the first check isn't necessary :+1: